getting-started/what-is-solidus.md

```diff
@@ -24,36 +24,32 @@ For maximum flexibility, you can decide you just want to install specific gems a
 
 ### In a new app
 
-If you don't have an existing Ruby on Rails application yet, simply create one:
+If you don't have an existing Ruby on Rails application yet, simply create one with sqlite3 as your database:
 
 ```bash
 $ rails new amazing_store --skip-javascript
 ```
+or for PostgreSQL as your database:
+
+```bash
+$ rails new amazing_store --skip-javascript --d=postgresql
+```
 
 Solidus doesn't require the JavaScript compiler shipped with Rails by default \(Webpacker\). You are still free to install it and use it in your store, though.
 
 Once you have generated your new Rails application, you can proceed as if you were installing Solidus [in an existing app](what-is-solidus.md#in-an-existing-app).
 
 ### In an existing app
 
-If you have an existing Ruby on Rails application, installing Solidus is fairly simple:
+If you have an existing Ruby on Rails application, installing Solidus is fairly simple. In your CLI:
 
-{% code title="Gemfile" %}
-```ruby
-# ...
-
-gem 'solidus'
-```
-{% endcode %}
-
-Then, install the bundle and configure Solidus:
 
 ```bash
-$ bundle install
+$ bundle add 'solidus' --version '~>3.0.0'
 $ bin/rails generate solidus:install
 ```
 
-The installer will prompt you for an admin email and password. You can leave the default \(email: admin@example.com, password: test123\) or enter your own.
+The installer will prompt you on a few questions before completing the installation. First it will ask if you would like to use the default authentication solution, [Devise](https://github.com/heartcombo/devise), or your implement your own. Next it will ask what payment service you would like to install with solidus. Currently, Solidus only supports Paypal and is the default option. You may install your own and avoid implementing Paypal by typing 'none' when prompted. Finally, you will be prompted for an admin email and password. You can leave the default \(email: admin@example.com, password: test123\) or enter your own.
```

Comment:
Instead of suggesting to install "your own payment method", we may want to link to the [extensions](https://solidus.io/extensions/) page, which contains a list of alternative payment methods in the Payment section. WDYT?

Reply:
> Instead of suggesting to install "your own payment method", we may want to link to the [extensions](https://solidus.io/extensions/) page, which contains a list of alternative payment methods in the Payment section. WDYT?

I Agree with this and have updated this line of text.